src/utils/Debounce.ts

```diff
@@ -5,12 +5,13 @@ export const debounce = <T>(
   reset: T,
   x: any,
   y: any,
+  z: any,
   timeout: any
 ) => {
 
   clearTimeout(timeout.current);
   timeout.current = setTimeout(() => {
-    func(x, y)
+    func(x, y, z)
```

Comment:
we might need to make an array of functions to call, so its one single parameter that is an array, vs keep adding parameters